### ISSUE TITLE
ctrl+a in quick open

### ISSUE THREAD

**Issue by joachov** [Microsoft/vscode#60270]
Issue Type: <b>Bug</b>

When using ctrl+a in the quick open box, it selects all in the active window. Not sure if this is a bug, but I would personally expect it to select all in the actual input box instead.

VS Code version: Code 1.28.0 (431ef9da3cf88a7e164f9d33bf62695e07c6c2a9, 2018-10-05T14:58:53.203Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz (8 x 2712)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.88GB (4.84GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
tslint|eg2|1.0.39
python|ms-|2018.8.0
PowerShell|ms-|1.9.0
debugger-for-chrome|msj|4.10.2

</details>

**Comment by scottymcraig** [comment]
This appears to be fixed in 1.29.0 (5bd6ebe2dbc4dee510699020a25f70a69784ce8c).  Pressing on Ctrl+a in the quick open only selects the text in the quick open edit box.

**Comment by sbatten** [comment]
this is specific to having web views open. I believe @mjbvz made a change to support select all in this context and maybe it is conflicting. This reproduces in stable and insiders

**Comment by mjbvz** [comment]
Yes this iteration we update most of the webview keybindings to apply when the webview is the active editor. Previously they only applied when the webview was focused.

Can we add an `when` clause that disables these webview keybindings when any sort of other input is active?

**Comment by bpasero** [comment]
> Can we add an when clause that disables these webview keybindings when any sort of other input is active?

Unfortunately we have many input boxes. I am not sure there is a DOM event we could use to set this context (I guess some focus/blur listener in capture phase could maybe do the trick?). 

Moving to October.

**Comment by bpasero** [comment]
Actually there is a `InputFocus` context key that we already use. However using it I found out that it does not seem to reset when the webview gets active. I wonder if that context key needs a `focusOut` handler as well to detect loosing focus.
